swagger-templates/src/main/java/com/okta/swagger/codegen/OktaJavaClientImplCodegen.java

```diff
@@ -211,7 +211,8 @@ public CodegenOperation fromOperation(String path,
                 co.vendorExtensions.put("dsMethod", "save");
             }
             if ("updateBrandTheme".equals(operation.getOperationId()) ||
-                "updateFeatureForApplication".equals(operation.getOperationId())) {
+                "updateFeatureForApplication".equals(operation.getOperationId()) ||
+                "updateEmailTemplateCustomization".equals(operation.getOperationId())) {
```

Comment:
Looks like some tests were removed for email templates. Do we need to add new ones?

Reply:
No tests were removed actually.

Reply:
You're right, I got my added vs removed mixed up locally. Sorry